src/ToolBox/superpmi/superpmi-shared/standardpch.h

```diff
@@ -109,3 +109,10 @@
 
 #define DEFAULT_REAL_JIT_NAME_A MAKEDLLNAME_A("clrjit2")
 #define DEFAULT_REAL_JIT_NAME_W MAKEDLLNAME_W("clrjit2")
+
+#if !defined(_MSC_VER) && !defined(__llvm__)
+static inline void __debugbreak()
+{
```

Comment:
You should be able to call DebugBreak() here, it is defined in PAL. Or just `#define __debugbreak DebugBreak`

Reply:
I added DebugBreak() here.